tests/test_schema.py

```diff
@@ -3789,13 +3790,19 @@ def test_describe_01(self):
 
             r"""
             function stdgraphql::short_name(name: std::str) -> std::str {
+                orig_nativecode :=
+                    r"SELECT (((name)[5:] IF (name LIKE 'std::%') ELSE
```

Comment:
It's unfortunate we can no longer retain the original source code of functions.

Reply:
We actually do store the original source, this is an artifact of parsing and codegening it.  Need more refactoring to avoid the roundtrip.